nav2-usability/scenario_code/lib/deeplink-queryparam/deeplink-queryparam_beamer.dart

```diff
@@ -49,10 +49,11 @@ class BooksApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp.router(
       title: 'Books App',
-      routerDelegate: BeamerRouterDelegate(
+      routerDelegate: BeamerDelegate(
+        transitionDelegate: NoAnimationTransitionDelegate(),
```

Comment:
Does the new version changes the default transition behavior, so this line needs to be added?

Reply:
`v0.13` doesn't change the default transition - it is still [DefaultTransitionDelegate](https://api.flutter.dev/flutter/widgets/DefaultTransitionDelegate-class.html), this is just a cosmetic change that looks better in this example.